themes/level_1/level1.py

```diff
@@ -16,7 +16,8 @@ def genHTMLLevel1(user_data, project_repos, hackathon_repos, blogs, stats_choice
     <img class="img-rounded" alt="Image" height="350px" src="{user_data['git_photo_url']}" />
     <h1 style='color:#f24b4b'> Hi, I am <a href="https://github.com/{user_data['username']}">{user_data['name']}</a>👨‍🎓 </h1>
     <h2 style='color:#21439e'>{user_data['git_bio']}</h2>
-    <!-- GITHUBSTATS-ENTRY -->  
+    <a href={resume_link} target="_blank"><h2 style='color:#d45131'>👉View Resume</h2></a>
```

Comment:
The href link needs to be in `""`.

Suggested change:
`"{resume_link}"`

Reply:
Have added the quotes.